### ISSUE TITLE
Settings cannot be set

### ISSUE THREAD

**Issue by hyperh** [arboleya/electrify#28]
I'm having trouble getting the Meteor settings to work.

`electrify -s settings.json` doesn't seem to work for me. I've double checked that `settings.json` is in the right path when I run the command.

**Comment by arboleya** [comment]
@hyperh Hi, I don't recall any remaining problems with the settings, but I'll check. Tks for reporting.

**Comment by hyperh** [comment]
I did a `console.log(Meteor.settings)` on the server side.
It returns `{ public: {} }`

My Meteor settings are supposed to be:

```
{
  "public": {
    "test": "thisisatest"
  },

  "embedlyApiKey": "mykey"
}
```

Update: Even when I change my `settings.json` to only

```
{
}
```

My `console.log`returns the same `{ public: {} }`

**Comment by arboleya** [comment]
Yes, because the `public` namespace inside the settings object is automatically filled in by Meteor. ;)

**Comment by coniel** [comment]
Also having the same issue, tried starting is using both `electrify -s settings.json` and `electrify --settings settings.json` and not getting any settings.

**Comment by arboleya** [comment]
I've found the problem, it only occurs when developing, not after packaging the app.

I'm on it, should release this fix tomorrow.

**Comment by arboleya** [comment]
Fixed in `2.1.3`, please update.

Closing, feel free to reopen in case something goes wrong.
